.github/FUNDING.yml

```diff
@@ -1,3 +1,3 @@
 # These are supported funding model platforms
-
-custom: https://numfocus.salsalabs.org/donate-to-matplotlib/index.html
+github: [numfocus]
+custom: https://numfocus.org/donate-to-matplotlib
```

Comment:
this link loads a page with the astropy logo

Reply:
that is, er, a bug. thanks for catching! will fix now.

Reply:
should work properly now; please refresh.

Reply:
Works, thanks for the quick turn around!